README.md

```diff
@@ -41,6 +41,26 @@ Where `[ICON SLUG]` is replaced by the [slug] of the icon you want to use, for e
 
 These examples use the latest major version. This means you won't receive any updates following the next major release. You can use `@latest` instead to receive updates indefinitely. However, this will result in a `404` error if the icon is removed.
 
+#### CDN with colors
+
+We also provide a CDN service which allows you to use colors.
+
+```html
+<img height="32" width="32" src="https://cdn.simpleicons.org/[ICON_SLUG]" />
+<img height="32" width="32" src="https://cdn.simpleicons.org/[ICON_SLUG]/[COLOR]" />
+```
+
+Where `[COLOR]` is optional, and can be replaced by the [hex triplet](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet) or [CSS keywords](https://www.w3.org/wiki/CSS/Properties/color/keywords) of the icon you want to you use. The color is defaulted to the color of icon from [Simpleicons.org](https://simpleicons.org). For example:
```

Comment:
Is not exactly true that the color can be only formatted using an HEX triplet. I would write that also accepts HEX cuatriplets because a fourth digit can be passed for an opacity value.

Reply:
Ah yes. Let me change the name to `hex color`, and change its link to https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color.

The [logic for parsing the hex color](https://github.com/LitoMore/simple-icons-cdn/blob/main/source/utils.ts#L3) is different from the hex color. I will fix that.

Reply:
Fixed within https://github.com/LitoMore/simple-icons-cdn/commit/9265817bdd0294e6fb9c5e8aba8db3d5c3b6c193.